<?xml version="1.0"?>
<launch>
<arg name="objects"/>
<arg name="simulation" default="1"/>
<arg name="gripper_enable" default="1"/>
<node name ="apriltag" pkg="hw4" type="apriltag" args="$(arg objects)"/>
<node name ="move" pkg="hw4" type="move" output="screen" ns="ur5">
<param name="simulation" type= "int" value="$(arg simulation)"/>
<param name="gripper_enable" type= "int" value="$(arg gripper_enable)"/>
<remap from="/ur5/tag_detections" to="/tag_detections"/>
<remap from="/ur5/tf" to="/tf"/>
</node>
</launch>
